src/components/AssetCard.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <v-card>
+  <v-card height="100%">
```

Comment:
Doing this would mean the footer will be just below the image rather than at the bottom